### ISSUE TITLE
Qubes OS GUI Tools should be tested on 1366x768 displays (or resolutions lower than FHD)

### ISSUE THREAD

**Issue by alimirjamali** [QubesOS/qubes-issues#9285]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
N/A - All
Screenshots from R4.2.1

### Brief summary
According to the statistics ([1](https://www.w3schools.com/browsers/browsers_display.asp), [2](https://www.designrush.com/agency/web-development-companies/trends/website-dimensions), [3](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide)), over 30% of global desktop users are using monitors with a resolution of 1366x768 or lower than FHD. This figure might be higher in Qubes OS user base as many might be using old generation laptops due to easy Intel ME bypass. Some of the current Qubes OS certified laptops (Nitrokey T430 & X230) are shipped with such displays.

It would be better to test GUI Tools on a low resolution display and assure they work well on such machines.

### Steps to reproduce
Open Qubes OS Updater on a display with a resolution of 1366x768. `CANCEL` & `UPDATE` buttons are off screen. This might be confusing without background knowledge. Fortunately you could maximize Updater window and get the buttons. Screenshot:
![Qubes OS Update GUI](https://github.com/QubesOS/qubes-issues/assets/5407317/1b95b071-2f2e-493f-b4a0-03929af81dbc)

Open Qubes OS policy editor. The 3 buttons are off screen. You can not get them by resizing or maximizing policy editor window. User should use the menu or hot-keys. Screenshot:
![Qubes OS Policy Editor](https://github.com/QubesOS/qubes-issues/assets/5407317/5510dc52-66ef-4ef1-8bd7-d1fc988b4fd0)

### Expected behavior
GUI Tools should appear well on both Half-HD and Full-HD displays.
### Actual behavior
GUI Tools are optimized for Full-HD displays.

**Comment by unman** [comment]
The x230 has a 1366x768 display.
Opening Qubes OS Updater produces a window with a scrollable list of
templates, and the `CANCEL` & `UPDATE` buttons are ON screen.
I've tested this in both KDE and Xfce. I dont know why you should see
different.

The policy editor is "off" in Xfce, I think. (in KDE the policy editor
doesnt appear in the KDE menu under Qubes Tools - something off in the
desktop file?)

**Comment by unman** [comment]
The policy editor is "off" in KDE. Hiding the panel from the bottom
of the screen allows the Top portions of the buttons to be seen.

**Comment by alimirjamali** [comment]
> I've tested this in both KDE and Xfce. I dont know why you should see different.

It would be probable because of the slightly resized XFCE panel bar (36 pixel). I believe most of other settings should be the default XFCE/Adwaita settings.
For better clarification, the tested `qubes-desktop-linux-manager` version is `4.2.23-1`

**Comment by marmarek** [comment]
FWIW, most integration tests (which doesn't mean all the components...) runs with 1024x768 in openQA. And yes, we do notice issues this way from time to time. For example, the policy editor issue is at https://github.com/QubesOS/qubes-issues/issues/9060, found exactly this way.

So, I guess the issue, as worded right now, is done already.

**Comment by marmarek** [comment]
This is the updater GUI at 1024x768: https://openqa.qubes-os.org/tests/102316#step/qui_widgets_update/10
It fits on the screen there just fine. Maybe it depends on some appearance settings (font size? theme?), or just number of templates?

**Comment by alimirjamali** [comment]
I did a sanity check and did `qubes-dom0-update --action=reinstall qubes-desktop-linux-manager`.

Then I made a quick side-by-side comparison in Krita. Higher number of templates might be the cause of this issue.

It is not a serious issue as it is easy to maximize the Updater and have the CANCEL & UPDATE buttons. Also most users will not have so many templates. Maybe it is better to forget about this issue and focus on more important issues.

![updater](https://github.com/QubesOS/qubes-issues/assets/5407317/6d27b106-040d-4c55-8ac8-91e0354f6f10)
